example/gardener-local/kind/cluster2-skaffold.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: kind.x-k8s.io/v1alpha4
```

Comment:
As suggested above, let's move this file to `example/gardener-local/kind2/cluster-skaffold.yaml`